From: "Vishal Moola (Oracle)" <vishal.moola@gmail.com>
Subject: mm: remove unmap_and_put_page()
Date: Wed, 9 Jul 2025 12:40:18 -0700

There are no callers of unmap_and_put_page() left. Remove it.

Link: https://lkml.kernel.org/r/20250709194017.927978-6-vishal.moola@gmail.com
Signed-off-by: Vishal Moola (Oracle) <vishal.moola@gmail.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Jordan Rome <linux@jordanrome.com>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/highmem.h |    6 ------
 1 file changed, 6 deletions(-)

--- a/include/linux/highmem.h~mm-remove-unmap_and_put_page
+++ a/include/linux/highmem.h
@@ -682,10 +682,4 @@ static inline void folio_release_kmap(st
 	kunmap_local(addr);
 	folio_put(folio);
 }
-
-static inline void unmap_and_put_page(struct page *page, void *addr)
-{
-	folio_release_kmap(page_folio(page), addr);
-}
-
 #endif /* _LINUX_HIGHMEM_H */
_
